Lightning tree component

HoltMichael opened the pull request:
Completed the lightning:verticalNavigation and lightning:tree

Also added a third dynamic component to flick between the 2 styles.
(First time committing - let me know if there's anything else I need to do)

rreboucas commented:
Hi Mike, 

Could you please add the following items to the Tree Navigation Components when you have a chance?

1. Add fixed white background to both components.
2. Add Header with a title like "Dynamic Navigation"
    Please look at example for both #1 and #2  on InterviewList.cmp

3. Add the following copyright comments to the top of each file in the bundle and related apex/vf etc:

xml style markup, svg, vf etc:

<!--

      Copyright (c) 2018, salesforce.com, inc.
      All rights reserved.
      SPDX-License-Identifier: BSD-3-Clause
      For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause

-->

js/apex/css:

/*
* Copyright (c) 2018, salesforce.com, inc.
* All rights reserved.
* SPDX-License-Identifier: BSD-3-Clause
* For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
*/

Thanks for your help with this new awesome component!

HoltMichael commented:
I've made the changes, tried to do a new pull request, but not sure it's worked? Should I be continuing on this chain? Not sure what the process is here!

With regard to the changes, I've added the Copyright in all the relevant places and added a background to all components. I've also added the banner to the dynamic component.
